config/puma.rb

```diff
@@ -0,0 +1,15 @@
+workers Integer(ENV['WEB_CONCURRENCY'] || 2)
+threads_count = Integer(ENV['MAX_THREADS'] || 5)
```

Comment:
Yes. I actually copied this config from [Heroku's documentation](https://devcenter.heroku.com/articles/deploying-rails-applications-with-the-puma-web-server), and this config is for Puma's concurrency setting and has nothing to do with dyno.